docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+      <title>Existing Tooling</title>
+      <para>
+        Today figuring out the true security of your hardware and firmware
+        requires sifting through the marketing documentation provided by the
+        OEM and in many cases just “trusting” they did it right.
+        Tools such as Chipsec can check the hardware configuration, but they do
+        not work out of the box and use technical jargon that an average user
+        cannot interpret.
+        Unfortunately, running a tool like Chipsec requires that you actively
+        turn off some security layers such as UEFI Secure Boot, and allow 3rd
+        party unsigned kernel modules to be loaded.
+      </para>
+    </refsect2>
+
+    <refsect1 id="verifying">
+      <title>Verifying Host Firmware Security</title>
+      <para>
+        To start out some core protections must be assigned a relative importance.
+        Then an evaluation must be done to determine how each vendor is conforming
+        to the model.
+        For instance, a user might say that for home use any hardware the bare
+        minimum security level (<code>HSI:1</code>) is <emphasis>good enough</emphasis>.
+        For a work laptop the company IT department might restrict the choice of
+        models to anything meeting the criteria of level <code>HSI:2</code> or
+        above.
+        A journalist or a security researcher would only buy level <code>HSI:3</code>
+        and above.
+        The reality is that <code>HSI:4</code> is going to be more expensive
+        than some unbranded hardware that is rated <code>HSI:0</code>.
+      </para>
+      <para>
+        To be trusted, this rating information should be distributed in a
+        centralized agnostic database such as the LVFS.
+      </para>
+      <para>
+        Of course, tools need to detect implementation errors, and to verify that
+        the model that is measured does indeed match the HSI level advertised by
+        the LVFS.
+        Some existing compliance solutions place the burden on the OEM to define
+        what firmware security has been implemented, which is easy to get wrong
+        and in some cases impossible to verify.
+      </para>
+      <para>
+        For this reason HSI will only measure security protections that can be
+        verified by the end user without requiring any extra hardware to be
+        connected, additional software to be installed, or disabling any existing
+        security layers to measure.
+      </para>
+    </refsect1>
+
+    <refsect2 id="runtime-behaviour">
+      <title>Runtime Behavior</title>
+      <para>
+        Orthogonal to the security features provided by the firmware there are
+        other security considerations related to the firmware which may require
+        internet access to discover or that runtime OS changes directly affect
+        the security of the firmware.
+        It would not make sense to have <emphasis>“have updates on the LVFS”</emphasis>
```

Comment:
with the docbook markup, does it still need `"`

Reply:
Good point, fixed.